File: services/device/src/main/java/com/example/iter/device/dto/response/MyEquipmentSummaryResponse.java
package com.example.iter.device.dto.response;

import com.example.iter.device.domain.entity.EquipmentCategory;
import com.example.iter.device.domain.entity.EquipmentStatus;
import com.example.iter.device.domain.entity.ProductConditionType;

import java.math.BigDecimal;
import java.time.LocalDate;

public record MyEquipmentSummaryResponse(
        Long id,
        String name,
        EquipmentCategory category,
        BigDecimal dailyPrice,
        EquipmentStatus status,
        ProductConditionType productCondition,
        String thumbnailUrl,
        LocalDate availableFrom,
        LocalDate availableTo
) {
}

File: services/device/src/main/kotlin/com/example/iter/device/dto/response/MyEquipmentSummaryResponse.kt
package com.example.iter.device.dto.response

import com.example.iter.device.domain.entity.EquipmentCategory
import com.example.iter.device.domain.entity.EquipmentStatus
import com.example.iter.device.domain.entity.ProductConditionType

import java.math.BigDecimal
import java.time.LocalDate

data class MyEquipmentSummaryResponse(
    val id: Long,
    val name: String,
    val category: EquipmentCategory,
    val dailyPrice: BigDecimal,
    val status: EquipmentStatus,
    val productCondition: ProductConditionType,
    val thumbnailUrl: String?,
    val availableFrom: LocalDate?,
    val availableTo: LocalDate?,
)
